src/Controllers/Library/search.php

```diff
@@ -1,6 +1,54 @@
 <?php
-
+/**
+ * Allows URY Librarians to search for Tracks.
+ */
 use \MyRadio\MyRadio\CoreUtils;
+use \MyRadio\MyRadio\URLUtils;
+use \MyRadio\ServiceAPI\MyRadio_Track;
+use \MyRadio\MyRadio\MyRadioForm;
+use \MyRadio\MyRadio\MyRadioFormField;
+
+$form = new MyRadioForm(
+    'lib_search',
+    'Library',
+    'search',
```

Comment:
Just looked at those, like this: https://github.com/UniversityRadioYork/MyRadio/blob/master/src/Classes/MyRadio/MyRadio_Availability.php#L366

It seems like you would only use that if it were inside a function? Or am I misunderstanding you?